rest_framework_simplejwt/models.py

```diff
@@ -19,11 +18,13 @@ class instead of a `User` model instance.  Instances of this class act as
     # inactive user
     is_active = True
 
-    _groups = EmptyManager(auth_models.Group)
-    _user_permissions = EmptyManager(auth_models.Permission)
 
     def __init__(self, token):
         self.token = token
+        from django.contrib.auth import models as auth_models
```

Comment:
Since this is always used, it's best if the import is still left at the top for performance.